### PR TITLE
Add Restlet and Rest.li

### PR DESCRIPTION
**Pull request by code-guru**
Add [reslet framework](http://restlet.com/projects/restlet-framework/)  and [Rest.li](http://rest.li) to the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -502,6 +502,8 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Jersey](https://jersey.java.net/) - JAX-RS reference implementation.
 * [RESTEasy](http://resteasy.jboss.org/) - Fully certified and portable implementation of the JAX-RS specification.
 * [RestExpress](https://github.com/RestExpress/RestExpress) - Thin wrapper on the JBOSS Netty HTTP stack to provide scaling and performance.
+* [Restlet](http://restlet.com/projects/restlet-framework/)) - Restlet Framework helps Java developers build better web APIs that follow the REST architecture style.
```
Comment (akullpp) [inline README.md]: Unfortunately, you have a double `)`. Description is too generic and doesn't say anything.

File: README.md [hunk 2]
```diff
@@ -502,6 +502,8 @@ A curated list of awesome Java frameworks, libraries and software.
...
+* [Rest.li](http://rest.li/) - Rest.li is an open source REST framework for building robust, scalable RESTful architectures using type-safe bindings and asynchronous, non-blocking IO.
```
Comment (akullpp) [inline README.md]: We use a rather condensed style for descriptions, e.g.

`Framework for building robust, scalable RESTful architectures using type-safe bindings and asynchronous, non-blocking IO.`

**Comment by Krasnyanskiy** [comment]
+1 for Restlet.

**Comment by akullpp** [comment]
Added rest.li in c7e2aa7 and Restlet in c7e2aa7

**Pull request closed without merging**